src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/pipeline/RemoteBuildPipelineStep.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2014 Jesse Glick.
```

Comment:
Huh?

Reply:
Oh, looks like a copy-paste error when we had a look at some of the other official pipeline steps. Missed that, maybe because the IDE collapses it...